### DISCUSSION TITLE
Talos OSD encryption native support

### DISCUSSION THREAD

**Discussion by samos667** [JJGadgets/Biohazard#858] [General]
Hi ! FYI, this [issue](https://github.com/siderolabs/talos/issues/3129) was fixed, so it's no longer needed to patch LVM config file.
https://github.com/siderolabs/talos/pull/7411

**Comment by JJGadgets** [comment]
Guy with Nagatoro profile pic!

Yeah I already removed the patch, just didn't close the issue haha, thanks for reminding!
